### ISSUE TITLE
Error while adding to meteor 1.4.1.1

### ISSUE THREAD

**Issue by toolX** [JackAdams/meteor-editable-text#77]
Hi! Thank you for the library. I have to add it because we use it during our Meteor Development course on  Coursera.
The problem is when I try to add your library to my project I get the following error:
{lamb} meteor add babrahams:editable-text
C:\Users\Phill\AppData\Local.meteor\packages\meteor-tool\1.4.1_1\mt-os.windows.x86_32\isopackets\ddp\npm\node_modules\meteor\promise\node_modules\meteor-promise\promise_server.js:165
      throw error;
      ^

Error: ENOENT: no such file or directory, open 'C:\Temp\mt-lrxeki\os\packages\gwendall_body-events.js.map'
    at Error (native)

Any ideas?
Thanks!

P.S. Sorry for nub question :)

**Comment by JackAdams** [comment]
I don't have a windows dev machine, so might not be able to reproduce this.  Will try with a fresh project on my mac.

If you could let me know a bit more about the project, that would help. Either:
1) public repo (this would be best for diagnosing problem)
OR
2) contents of .meteor/packages file AND link to the coursera project

**Comment by JackAdams** [comment]
Just tried adding it to a fresh 1.4.1.1 install on a mac and it worked fine.

It might be a problem with the windows version of the meteor build tool or there may be something unusual with your particular dev setup.

Try `meteor add gwendall:body-events` first and see what happens.

You could also try `meteor add gwendall:impersonate` as an experiment (as this is another package with a dependency on `gwendall:body-events`).  This might reveal whether it is a problem with the `babrahams:editable-text` package or a wider problem with your dev setup.

**Comment by toolX** [comment]
You are right! Here is a link to my github [repository](https://github.com/toolX/textcircle).

When I add gwendall:body-events it seems like I receive the same error:

{lamb} meteor add gwendall:body-events
C:\Users\Phill\AppData\Local.meteor\packages\meteor-tool\1.4.1_1\mt-os.windows.x86_32\isopackets\ddp\npm\node_modules\meteor\promise\node_modules\meteor-promise\promise_server.js:165
      throw error;
      ^

Error: ENOENT: no such file or directory, open 'C:\Temp\mt-82kuoz\os\packages\gwendall_body-events.js.map'
    at Error (native)

The same situation with adding gwendall:impersonate :(

Anyway, thank you for your time and advices - I really appreciate your help!

**Comment by JackAdams** [comment]
Not sure what's going on in your dev setup, but I cloned your repo, added the package and changed `{{title}}` to `{{> editableText collection="documents" field="title"}}` and now the doc name is editable:
![screen shot 2016-09-11 at 11 01 22 pm](https://cloud.githubusercontent.com/assets/1647779/18418159/b5a96f30-7873-11e6-9eeb-8eb9a7e3ed73.png)

**Comment by toolX** [comment]
Ok. I just can't add the package. 
Thank you anyway!

**Comment by toolX** [comment]
I've decided to continue digging this problem. I've created a new app with meteor version 1.2.1 and install your package - all works just fine! And I've also tried a 1.3.0 version, too. Without any problem. Moreover I've then updated meteor to the latest version 1.4.1.1 - your package works perfect.
So I don't think that it is a problem with my system. Maybe it is a problem with dependencies?
By the way, I use Windows 10.
